conda-store-server/conda_store_server/schema.py

```diff
@@ -9,6 +9,8 @@
 from conda_store_server import conda_utils, utils
 from pydantic import BaseModel, Field, ValidationError, constr, validator
 
+ON_WIN = sys.platform.startswith("win")
```

Comment:
This is already declared in another file. Can we move that definition somewhere where it makes the most sense and import everywhere? Ideally, this should be done in a way that minimizes potential import cycles.